subdomains.json

```diff
@@ -143,6 +143,7 @@
   "tolis": "tolisdev.xyz",
   "tudor": "artsui.github.io",
   "uki": "uki.vercel.app",
+  "unseen": "unseenacoustics.github.io"
```

Comment:
Looks like you missed `,`, correct json format should be `"unseen": "unseenacoustics.github.io",`